config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes-sigs/kubebuilder:
+
+    - name: post-kube-rbac-proxy-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-kubebuilder-kube-rbac-proxy, sig-k8s-infra-gcb
```

Comment:
@rikatz 

It is falling here `could not apply prowjob annotations: couldn't find dashboard "sig-kubebuilder-kube-rbac-proxy" for job "post-kube-rbac-proxy-push-images"`

Should we not add the dashboard (remove sig-kubebuilder-kube-rbac-proxy from the above line ) or is that something that we need to create? OR indeed we should add another value?

Reply:
you should probably change it to "sig-api-machinery-kubebuilder" or add a new one in https://github.com/kubernetes/test-infra/blob/a479c206796fee9ded9793bfffcd1ca93b87b35f/config/testgrids/kubernetes/sig-api-machinery/config.yaml